src/init/application/templates/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "<%= name %>",
+  "short_name": "<%= name %>",
+  "description": "<%= description %>",
```

Comment:
need an if statement to make sure description isn't empty string when not provided